### ISSUE TITLE
Update Translation Report Color Scheme

### ISSUE THREAD

**Issue by kinson** [elixirschool/school_house#123]
# Overview

Right now there are a lot of accessibility issues with the translation report page related to insufficient color contrast.

The colors used here should be updated to be generally more readable and ideally fix the accessibility issues.

# Screenshots
<img width="1440" alt="Screen Shot 2021-08-07 at 4 06 58 PM" src="https://user-images.githubusercontent.com/1526888/128614993-063ac9e3-4b79-451e-b29f-d4d5b9528a85.png">
<img width="1439" alt="Screen Shot 2021-08-07 at 4 07 07 PM" src="https://user-images.githubusercontent.com/1526888/128614995-3249d583-5099-4eac-8475-83a48e214054.png">

**Comment by kinson** [comment]
Closed by #124
